win32ss/user/ntuser/ime.c

```diff
@@ -1957,7 +2053,7 @@ BOOL FASTCALL IntImeCanDestroyDefIME(PWND pImeWnd, PWND pwndTarget)
     }
     _SEH2_EXCEPT(EXCEPTION_EXECUTE_HANDLER)
     {
-        ;
+        NOTHING;
```

Comment:
Note that if the pimeui were to be used later in the code, in the future, then this would have to be certainly replaced by e.g. a return FALSE or something like that.

Reply:
I see.